src/accessibility/index.ts

```diff
@@ -239,7 +239,7 @@ async function toPassAxeTests(
   } catch {
     throw removeFuncFromStackTrace(
       new Error(
-        'Install @axe-core/puppeteer and axe-core to use the toPassAxeTests matcher',
+        'Install @axe-core/puppeteer to use the toPassAxeTests matcher',
```

Comment:
I realized that @axe-core/puppeteer comes with its own version of axe-core so it is not necessary for users to directly install axe-core.